src/components/Calendar/doubleCalendar/styled/select.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { FONT_SIZE_HEADING_MEDIUM, FONT_SIZE_TEXT_LARGE } from '../../../../styles/fontSizes';
+import attachThemeAttrs from '../../../../styles/helpers/attachThemeAttrs';
+import Select from '../../../Select';
+
+const StyledSelect = attachThemeAttrs(styled(Select))`
```

Comment:
Since this select looks and work different to our Select component, I think we don't would use our Select component, we should style the native select
This way we avoid doing introspection in the Select component